### ISSUE TITLE
DEQ Spills Salesforce Widget Rebuild

### ISSUE THREAD

**Issue by stdavis** [agrc/planning-queue#501]
### Benefit

The Esri JavaScript API that the custom application relies on reached its end of support on July 1, 2024. While the application is expected to continue to operate without disruption, UGRC can no longer guarantee reliable support, enhancements, or long-term functionality. To address this, the application will need to be rebuilt using a new API. As part of this rebuild, we also plan to incorporate additional requested enhancements.

### Acceptance Criteria

- [ ] 

### Notes

- 

### Risks

- 

### Issue Reference

- #

**Comment by steveoh** [comment]
We have a meeting with Barry B on 1/8 to discuss preferred architecture

**Comment by stdavis** [comment]
### FY25 Q3 Sprint 2
- Upgraded app to latest atlas template and am almost done upgrading salesforce project to the latest recommendations

**Comment by stdavis** [comment]
## FY25 Q3 Sprint 4
- finalize project structure
- enable mouse wheel zooming
- flesh out basic map layout
- add reference layers and legend
- complete first successful push to salesforce sandbox (had to work with Barry on CSP)
- went down [major rabbit hole](https://github.com/agrc/kitchen-sink/pull/421) in kitchen sink. Hope to get this done in the next few days

**Comment by stdavis** [comment]
## FY25 Q3 Sprint 5

Finished up all of the items that I had pegged for this quarter. [These release notes](https://github.com/agrc/deq-spills/releases/tag/v3.0.0-3) cover it pretty well.

**Comment by stdavis** [comment]
## FY25 Q4 Sprint 1
- https://github.com/agrc/deq-spills/releases/tag/v3.0.0-5

Updates to eid skid and interactive map related to this project:
- Repointed the "Spill ID" field to pull data from the "CaseNumber" Salesforce field. This was previously pulling from the "Id" Salesforce field.
- Added a new "Salesforce ID" field to pull data from the "Id" Salesforce field.
- Added a "Report" field that contains a link to the report PDF in Salesforce.
- Added the "Report" field to the query results grid, the attributes table, and the "Additional Information" link
- Removed the Chemicals and Impacted Media related tables (this information is now available in the Salesforce PDF report)

**Comment by stdavis** [comment]
@chriswnek This project will carried over into the next FY. Do you want me to create a new issue or pull this one forward?

**Comment by chriswnek** [comment]
Since there's a lot of history and detail in this issue, let's just carry it over...but we can wait to actually move it to next quarter until after the PI Planning meeting on 6/30.

**Comment by stdavis** [comment]
## FY26Q1 Sprint 4
Developed a plan and started working on https://github.com/agrc/deq-spills/issues/129

**Comment by stdavis** [comment]
## FY26 Q1 Sprint 5

Got a good start on https://github.com/agrc/deq-spills/issues/129. Submitted https://github.com/agrc/deq-spills/pull/165 for Steve's review just to make sure that I'm headed in the right direction. I'll also need input from the client on the storage solution for these geometry (likely AGOL).

Once the PR is merged, this project will likely sit until they have a salesforce person ready to go.
